docs/code_conventions/README.MD

```diff
@@ -150,12 +368,12 @@ For example:
 Wherever possible, we try to link context together. This will prevent a command from overwriting existing data, or from creating duplicate entries in the context. To do this, observe the following:
```

Comment:
In outputs I would add:
Make sure to adhere to our [context standards](https://github.com/demisto/content/tree/master/docs/context_standards) when naming indicator outputs.